versioned_docs/version-8.3/apis-tools/tasklist-api/enums/sort.mdx

```diff
@@ -60,4 +60,4 @@ enum Sort {
 
 ### Member of
 
-[`TaskQuery`](/docs/apis-tools/tasklist-api/inputs/task-query.mdx) <Badge class="secondary" text="input"/>
```

Comment:
😬 yikes, there were so many of these, thank you for fixing all of them! I would have ended up modifying all of these at the time of archiving each of these versions so I owe you a coffee or something for doing it first.

Reply:
Easy peasey with Cmd+D 😆